lua/telescope/actions/init.lua

```diff
@@ -345,8 +345,6 @@ actions.git_checkout = function(prompt_bufnr)
   end
 end
 
---- Tell git to track the currently selected remote branch in Telescope
```

Comment:
can you just change these to be `-- comment here` rather than `---` with a todo that says we can add these back in when the upstream issue is fixed?

Reply:
Yup! Happy to do that!